guides/docs/controllers.md

```diff
@@ -452,7 +452,8 @@ The correct way to render the 404 page from `HelloWeb.PageController` is:
 def index(conn, _params) do
   conn
   |> put_status(:not_found)
-  |> render(HelloWeb.ErrorView, "404.html")
+  |> put_view(HelloWeb.ErrorView)
```

Comment:
Should we add a "Setting the view" section to introduce `put_view` and `plug :put_view, View` before using it?